### PR TITLE
Feature/sc 314

### PR DESCRIPTION
**Pull request by DanCh11**
- Created tests for LDA classes and character remover
 - Published HTML version of LDA notebook based on pwdb noun phrases ( waiting for feedback )

### CODE REVIEW

File: sem_covid/entrypoints/notebooks/topic_modeling/topic_modeling_wrangling/topic_visualizer.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 
+from typing import List
 from PIL.Image import Image
```
Comment (costezki) [inline sem_covid/entrypoints/notebooks/topic_modeling/topic_modeling_wrangling/topic_visualizer.py:3]: what is PIL? a new library?

File: tests/unit/test_data_cleaner.py [hunk 2]
```diff
@@ -74,3 +74,29 @@ def test_data_cleaner():
...
+def test_clean_text_from_specific_characters(transformed_pwdb_dataframe):
+    pwdb_descriptive_data = transformed_pwdb_dataframe['title'].map(str) + ' ' + \
+                            transformed_pwdb_dataframe['background_info_description'].map(str) + ' ' + \
+                            transformed_pwdb_dataframe['content_of_measure_description'].map(str) + ' ' + \
+                            transformed_pwdb_dataframe['use_of_measure_description'] + ' ' + \
+                            transformed_pwdb_dataframe['involvement_of_social_partners_description']
+    unused_characters = ["\\r", ">", "\n", "\\", "<", "''", "%", "...", "\'", '"', "(", "\n", "[", "]"]
+    clean_text = clean_text_from_specific_characters(pwdb_descriptive_data, unused_characters)
+
+    assert "\\r" not in clean_text
```
Comment (costezki) [inline tests/unit/test_data_cleaner.py:88]: these lines shall be reduced to a single assert. Investigate how to do it in a pythonic way.

Comment (costezki) [reply tests/unit/test_data_cleaner.py:88]: perhaps any() or all() functions can be of help

**Review by costezki — APPROVED** [review]

File: sem_covid/entrypoints/notebooks/topic_modeling/pwdb_single_words.ipynb [hunk 3]
```diff
@@ -47,6 +47,8 @@
     "import warnings\n",
```
[the remark this answers is not in the document]
Comment (DanCh11) [reply sem_covid/entrypoints/notebooks/topic_modeling/pwdb_single_words.ipynb:47]: <p>I rechecked it and made a mistake. This function is not working correctly. The idea was that a noun phrase can represent multiple topics, and basically this table should represent the topic and the number of how often the noun phrases represents them. For example the topic nr.1 is represented by 48 noun phrases.</p><p> </p>

[the remark this answers is not in the document]
Comment (DanCh11) [reply sem_covid/entrypoints/notebooks/topic_modeling/pwdb_single_words.ipynb:47]: <p>I think I will delete both tables. After rechecking both results, i saw that something went wrong, or i misspelled somewhere.</p>

File: sem_covid/entrypoints/notebooks/topic_modeling/topic_modeling_wrangling/topic_visualizer.py (hunk 1 above)
Comment (DanCh11) [reply sem_covid/entrypoints/notebooks/topic_modeling/topic_modeling_wrangling/topic_visualizer.py:3]: PIL it's python image library. It's integrated in python

File: tests/unit/test_data_cleaner.py (hunk 2 above)
Comment (DanCh11) [reply tests/unit/test_data_cleaner.py:88]: I tried to figure out how to reduce it into a single line, but the test expects to return a string not a bool or a list

TypeError: 'in <string>' requires string as left operand, not list -> using the list
TypeError: 'in <string>' requires string as left operand, not bool -> using all() or any()

**Pull request closed without merging**